consensus/msgs.go

```diff
@@ -7,81 +7,82 @@ import (
 	"github.com/cosmos/gogoproto/proto"
 
 	cmtcons "github.com/cometbft/cometbft/api/cometbft/consensus"
-	cmtproto "github.com/cometbft/cometbft/api/cometbft/types"
+	cmtproto "github.com/cometbft/cometbft/api/cometbft/types/v1"
 	cstypes "github.com/cometbft/cometbft/consensus/types"
 	"github.com/cometbft/cometbft/libs/bits"
 	cmtmath "github.com/cometbft/cometbft/libs/math"
 	"github.com/cometbft/cometbft/p2p"
 	"github.com/cometbft/cometbft/types"
 )
 
-// Takes a consensus message type and returns the proto defined consensus message,
-// wrapped in the discriminating Message container.
-func MsgToWrappedProto(msg Message) (cmtcons.Message, error) {
-	pb := cmtcons.Message{}
+// MsgToProto takes a consensus message type and returns the proto defined consensus message.
+//
+// TODO: This needs to be removed, but WALToProto depends on this.
```

Comment:
Can we open an issue on this?

Reply:
Sure, however this comment is present in main, I just merged it here.